addons/essentials/README.md

```diff
@@ -51,4 +52,4 @@ module.exports = {
 };
 ```
 
-Valid addon keys include: `actions`, `backgrounds`, `controls`, `docs`, `viewport`.
+Valid addon keys include: `actions`, `backgrounds`, `controls`, `docs`, `viewport`, `toolbars`.
```

Comment:
This is duplicated on the essentials intro in the docs. Should we dedupe?

Reply:
Yeah, mind taking that?